### PR TITLE
Circle CI

### PR DESCRIPTION
**Pull request by Xorima**
Added: License file
Added: Delivery config
Added: Circleci config
Added: Dangerfile
Added: spec tests
Fixed: Linting errors relating to hashes
Fixed: metadata

### CODE REVIEW

**Comment by kitchen-porter** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Please include a <a href="https://github.com/sous-chefs/dhcp/blob/master/CHANGELOG.md">CHANGELOG</a> entry.</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This Pull Request is probably missing tests.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: metadata.rb [hunk 1]
```diff
@@ -1,9 +1,13 @@
-name 'dnsmasq'
-maintainer 'Chris Roberts'
-maintainer_email 'chrisroberts.code@gmail.com'
-license 'Apache 2.0'
-description 'Installs and configures dnsmasq'
-version '0.2.0'
+name              'dnsmasq'
+maintainer        'Chris Roberts'
```
Comment (damacus) [inline metadata.rb]: maintainer        'Sous-Chefs'

File: metadata.rb [hunk 2]
```diff
@@ -1,9 +1,13 @@
...
+maintainer_email  'chrisroberts.code@gmail.com'
```
Comment (damacus) [inline metadata.rb]: maintainer_email  'help@sous-chefs.org'

File: metadata.rb [hunk 3]
```diff
@@ -1,9 +1,13 @@
...
+license           'Apache-2.0'
+description       'Installs and configures dnsmasq'
+long_description  IO.read(File.join(File.dirname(__FILE__), 'README.md'))
+chef_version      '>= 13.0' if respond_to?(:chef_version)
```
Comment (damacus) [inline metadata.rb]:
```suggestion
chef_version      '>= 13.0'
```

**Pull request merged**